components/dropdown/src/auro-dropdown.js

```diff
@@ -236,6 +277,10 @@ export class AuroDropdown extends LitElement {
     if (changedProperties.has('mobileFullscreenBreakpoint')) {
       this.bibContent.mobileFullscreenBreakpoint = this.mobileFullscreenBreakpoint;
     }
+
+    if (changedProperties.size === 0) {
```

Comment:
What does this do exactly?

Reply:
this is the condition to know if `updated` is called by `requestUpdate`